docs/Users_Guide/release-notes.rst

```diff
@@ -48,355 +48,212 @@ describes the bugfix, enhancement, or new feature.
-
-     * WaveletStat
-       (`#2252 <https://github.com/dtcenter/METplus/issues/2252>`_)
-
-
-  .. dropdown:: New Use Cases
-
-     NONE
-
-  .. dropdown:: Documentation
-
-     * **Add upgrade instructions for removing user wrapped MET config files**
-       (`#2349 <https://github.com/dtcenter/METplus/issues/2349>`_)
-     * Reorder Python Wrappers - MET Configuration tables to match order in wrapped MET config file
-       (`#2405 <https://github.com/dtcenter/METplus/issues/2405>`_)
-     * Enhancement to Difficulty Index use-case documentation
-       (`#2123 <https://github.com/dtcenter/METplus/issues/2123>`_)
-     * Modify the Documentation Overview section in the Contributor's Guide to add Conventions
-       (`#1667 <https://github.com/dtcenter/METplus/issues/1667>`_)
-     * Specify available tags on DockerHub
-       (`#2329 <https://github.com/dtcenter/METplus/issues/2329>`_)
+       (`#2489 <https://github.com/dtcenter/METplus/issues/2489>`_)
+     * Add the METplus development timeline for the next official release to the User's Guide
+       (`#2530 <https://github.com/dtcenter/METplus/issues/2530>`_)
+     * Update links in Verification Datasets Guide
+       (`#2584 <https://github.com/dtcenter/METplus/issues/2584>`_)
+     * Document SonarQube for the METplus components in the Contributor's Guide
+       (`#2666 <https://github.com/dtcenter/METplus/issues/2666>`_)
+     * Update the User Support section in the Contributor's Guide
+       (`#2679 <https://github.com/dtcenter/METplus/issues/2679>`_)
+     * Move the Existing Builds page to Read the Docs
+       (`#2716 <https://github.com/dtcenter/METplus/issues/2716>`_)
+     * Update release guide instructions for METplus Analysis components
+       (`#2737 <https://github.com/dtcenter/METplus/issues/2737>`_)
+     * Update Release Notes to include updating the schedule for releases
+       (`#2751 <https://github.com/dtcenter/METplus/issues/2751>`_)
+     * Update Release Guide to create the main_vX.Y branch for the first release candidate
+       (`#2754 <https://github.com/dtcenter/METplus/issues/2754>`_)
+     * Add allow missing input variables to glossary
+       (`#2757 <https://github.com/dtcenter/METplus/issues/2757>`_)
+     * Update the Existing Builds page
+       (`#2763 <https://github.com/dtcenter/METplus/issues/2763>`_)
+     * **Add instructions for running a use case using Apptainer**
+       (`#2772 <https://github.com/dtcenter/METplus/issues/2772>`_)
+     * Add a dedication to all of the User's Guides
+       (`#2780 <https://github.com/dtcenter/METplus/issues/2780>`_)
+     * **Include information on how to use command line arguments to override single config variables**
+       (`#2814 <https://github.com/dtcenter/METplus/issues/2814>`_)
 
   .. dropdown:: Internal
 
-     * Improve METplus test coverage
-       (`#2253 <https://github.com/dtcenter/METplus/issues/2253>`_)
      * Documentation: Make Headers Consistent in METplus components User's Guides
```

Comment:
Recommend moving to the Documentation section and removing the "Documentation: " prefix.